src/web3/wallet/ZedSignerStorage.ts

```diff
@@ -0,0 +1,13 @@
+import { ReactNativeSignersStorage } from '@usecapsule/react-native-wallet'
+
+export class ZedSignerStorage extends ReactNativeSignersStorage {
+  public async addAccount(account: string): Promise<void> {
+    await super.addAccount(account)
+    // @todo Store account in redux
+  }
+
+  public async getAccounts(): Promise<string[]> {
+    // @todo Retrieve accounts from redux
```

Comment:
Create a JIRA ticket to ensure that accounts can be retrieved from the local redux store.

Reply:
Created in [KW-216](https://kolektivo.atlassian.net/browse/KW-216?atlOrigin=eyJpIjoiZjNiNzU5OWFhODQ1NDU4ZjhiNjI1NzEzNWRiMTE4NjYiLCJwIjoiaiJ9)